typescript/sdk/src/providers/transactions/schemas.ts

```diff
@@ -0,0 +1,7 @@
+import { z } from 'zod';
+
+export const PopulatedTransactionSchema = z.object({
```

Comment:
I expect we'll want some kind of differentiation by protocol type here eventually, like what we do for `TypedProvider`s. Not blocking